nml/actions/action2var.py

```diff
@@ -819,6 +823,46 @@ def parse_minmax(value, unit_str, action_list, act6, offset):
 return_action_id = 0
 
 
+def create_ternary_action(guard, expr_true, expr_false, action_list, feature, var_scope):
+    act6 = action6.Action6()
+
+    var_range = 0x8A if get_scope(feature, 0x8A) == var_scope else 0x89
```

Comment:
`var_scope` will be the same for parent and self scope in case of vehicles, so converting `var_scope` back to `type_byte`/`var_range` won't work. `type_byte` needs to be passed separately from somewhere.